core/storage/statistics/gae_models.py

```diff
@@ -1273,8 +1273,8 @@ def get_model(
             exp_version: int. Version of the exploration.
 
         Returns:
-            ExplorationStatsModel. Exploration analytics model instance in
-            datastore.
+            ExplorationStatsModel|None. Exploration analytics model instance in
+            datastore or None.
```

Comment:
...in datastore, or None if no such model instance exists.

Reply:
Done!!